packages/broken-link-checker/README.md

```diff
@@ -0,0 +1,25 @@
+# Docs Kit Broken Link Checker
+
+This package provides an executable script that checks a statically built website on the file system for any broken internal or external links.
+
+It's based on the excellent [`broken-link-checker`](https://github.com/stevenvachon/broken-link-checker#readme) library and is fully preconfigured.
+
+The return code is the number of errors found, so it can be used directly in build scripts to have the build fail or pass.
```

Comment:
I'm not sure this is a good thing. Why not simply returning `process.exit(1)`?

Reply:
Right, that was just a lazy variant, will change that.